legend-engine-pure-code-compiled-core/src/main/resources/core/pure/corefunctions/dateExtension.pure

```diff
@@ -589,4 +589,10 @@ function meta::pure::functions::date::toEpochValue(d:Date[1]):Integer[1]
 function meta::pure::functions::date::toEpochValue(d:Date[1], unit:DurationUnit[1]):Integer[1]
 {
    dateDiff(%1970-01-01T00:00:00.000+0000, $d, $unit);
-}
\ No newline at end of file
+}
+
+function meta::pure::functions::date::validateDateTimeFormat(dateTimeFormat:String[1]):Boolean[1]
```

Comment:
Are we restricting to only few possible timezone's?

Reply:
Yes, we are restricting to the ones that are formattable by PureDate using regex

Reply:
So we can only understand couple of timezones?

Reply:
Updated the regex to all possible timezones